### ISSUE TITLE
CRITICAL:root:twint.run:Twint:Feed:noDataExpecting

### ISSUE THREAD

**Issue by tibor** [twintproject/twint#635]
# Issue Template
Please use this template!

### Initial Check

>Make sure you've checked the following:

- [x] Python version is 3.6;
- [x] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [x] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
```
userhandles = ['some', 'userhandles']
for userhandle in userhandles:
    print('\n-----------------\n\n{}'.format(userhandle))
    tweets = []
    c = twint.Config()
    c.Username = userhandle
    c.Store_object = True
    c.Store_object_tweets_list = tweets
    c.Hide_output = True

    twint.run.Search(c)
    a = b = 0
    for tweet in tweets:
        sql_insert_tweet = '''INSERT IGNORE INTO (..)"
        sql_insert_tweet_data = (..)
        a += 1
        print(a)
        try:
            mysql_cursor.execute(sql_insert_tweet, sql_insert_tweet_data)
            mysql_conn.commit()
            b += 1
        except Exception as e:
            print(e)
    print('{} / {}\n-----------------------\n'.format(b, a))
```

### Description of Issue
Twint would return following error:
`CRITICAL:root:twint.run:Twint:Feed:noDataExpecting value: line 1 column 1 (char 0)`

Example output:
```
-----------------

andrewkneebone
CRITICAL:root:twint.run:Twint:Feed:noDataExpecting value: line 1 column 1 (char 0)
CRITICAL:root:twint.run:Twint:Feed:noDataExpecting value: line 1 column 1 (char 0)
CRITICAL:root:twint.run:Twint:Feed:noDataExpecting value: line 1 column 1 (char 0)
CRITICAL:root:twint.run:Twint:Feed:noDataExpecting value: line 1 column 1 (char 0)
1
2
3
4
..
372
373
373 / 373
-----------------------
```

Usernames where this occured:
Piper
crystale
steverubel
andrewkneebone

### Environment Details
Ubuntu, Python 3.6.9

**Comment by pielco11** [comment]
Hi @tibor 

This issue has been already reported and there's already an open (and pinned) one #604

**Comment by tibor** [comment]
Sorry, didn’t see the pinned ones. Thanks!

**Comment by UpasanaDutta98** [comment]
I was facing this issue when I travelled from Boulder to India. I tried many different stuff, but I kept facing this error in India. When I was back in Boulder, I ran the same piece of code again, and guess what? I ran without any errors!

So I feel that at least in my case, it was some regional setting of the network w.r.t accessing Twitter data which was causing this error, I can't say in detail though.
